includes/wf_crm_webform_postprocess.inc

```diff
@@ -1896,8 +1898,9 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
     // tax integration
     // TODO: needs review if this is changed in 4.6
     if (!is_null($this->tax_rate)) {
-      $params['non_deductible_amount']  = $this->totalContribution;
-      $params['tax_amount'] = round(($params['total_amount'] / ($this->tax_rate + 100)) * $this->tax_rate, 2);
+      $params['non_deductible_amount']  = $totalContributionAmount;
+      $taxAmount = ($params['total_amount'] / ($this->tax_rate + 100)) * $this->tax_rate;
+      $params['tax_amount'] = round($taxAmount, 2);
     }
     $params['description'] = t('Webform Payment: @title', array('@title' => $this->node->title));
```

Comment:
Ok so I think all we need in this loop is:
`$params['tax_amount'] = round(($params['total_amount'] / ($this->tax_rate + 100)) * $this->tax_rate, 2);`